### ISSUE TITLE
Custom Loss Function

### ISSUE THREAD

**Issue by phildue** [pjreddie/darknet#860]
Hi,

Is it possible to load/transfer weights from tensorflow/keras to darknet?
If not how is it possible to use a custom loss function within darknet? And how would I go about implementing it?

Thanks!
